micarray/modal/angular.py

```diff
@@ -153,3 +195,23 @@ def grid_gauss(n):
     weights = np.repeat(weights, 2*n+2)
     weights *= np.pi / (n+1)      # sum(weights) == 4pi
     return azi, elev, weights
+
+
+def grid_equal_polar_angle(M, phi0=0):
```

Comment:
The spherical grid functions take the maximum `N` as the only argument.
For consistency I'd prefer the same behavior. (i.e. pass `N` and get `2N+1` positions.)

Reply:
@narahahn prefers the current version, being more useful in practice.
I prefer the less flexible but more consistent interface.

@spors What's your opinion?